StripeFinancialConnections/StripeFinancialConnections/Source/Common/HostController.swift

```diff
@@ -62,11 +63,13 @@ class HostController {
         clientSecret: String,
         returnURL: String?,
         publishableKey: String?,
-        stripeAccount: String?
+        stripeAccount: String?,
+        v1AnalyticsClient: STPAnalyticsClientProtocol = STPAnalyticsClient.sharedClient
```

Comment:
this is not too important but I feel like its cleaner: 

I think you can avoid accessing the `sharedClient` here, and instead just pass the correct value from `FinancialConnectionsSheet` (which has this as an instance value):

https://github.com/stripe/stripe-ios/blob/9c3d6eb1/StripeFinancialConnections/StripeFinancialConnections/Source/FinancialConnectionsSheet.swift#L207-L213

Reply:
Done!